recipes-bsp/u-boot/u-boot-distro-boot/uEnv.txt.in

```diff
@@ -108,6 +112,116 @@ bootcmd_boot=if test ${bootscript_debug} != 1; then \
+                           echo "DRY RUN: fuse cmp"; \
+                       else \
+                           fuse cmp ${fuse_bank} ${fuse_word} ${fuse_val}; \
+                       fi; \
+                       if test $? != "0"; then \
+                           env set fuse_fail 1; \
+                       fi; \
+                   fi ; \
+                   setexpr fuse_word $fuse_word + 1; \
+                   if test "${soc}" = "imx8m" || test "${soc}" = "imx7d"; then \
+                       if test "${fuse_word}" = "4"; then \
+                           setexpr fuse_bank $fuse_bank + 1; \
+                           setexpr fuse_word 0; \
+                       fi; \
+                   fi; \
+                   if test "${fuse_word}" = "72a"; then \
+                       env set fuse_word 730; \
+                   elif test "${fuse_word}" = "73a"; then \
+                       env set fuse_word 740; \
+                   fi; \
+               done; \
+               @@STATUS_CMD@@; \
+               if test $? != "0"; then \
+                   env set fuse_fail 1; \
+               fi; \
+               if test "${fuse_prog_close}" = "1" && test "${fuse_fail}" != "1"; then \
+                   if test "${fuse_dry_run}" = "1"; then \
+                       echo "DRY RUN: Closing is set"; \
+                   else \
+                       @@CLOSE_CMD@@; \
+                   fi; \
+                   if test $? != "0"; then \
+                       env set fuse_fail 1; \
+                   fi; \
+               fi; \
+               env set fuse_bank @@FUSE_BANK@@; \
+               env set fuse_word @@FUSE_WORD@@; \
+               env set save_vars 1; \
+           fi || true
+
+read_fuses=if test -n "${fuse_num}"; then \
+               for index in "${fuse_num}"; do \
+                   if env exists fuse_val_${index}; then \
+                       env set temp fuse cmp ${fuse_bank} ${fuse_word} 0x\\$fuse_val_$index; \
+                       run temp; \
+                   else \
+                       false; \
+                   fi; \
+                   if test $? != "0"; then \
+                       fuse readm ${fuse_bank} ${fuse_word} ${loadaddr}; \
+                       setexpr.l fuse_val_${index} "*${loadaddr}"; \
+                       env set save_vars 1; \
+                   fi; \
+                   setexpr fuse_word $fuse_word + 1; \
+                   if test "${soc}" = "imx8m" || test "${soc}" = "imx7d"; then \
+                       if test "${fuse_word}" = "4"; then \
+                           setexpr fuse_bank $fuse_bank + 1; \
+                           setexpr fuse_word 0; \
+                       fi; \
+                   fi; \
+                   if test "${fuse_word}" = "72a"; then \
+                       env set fuse_word 730; \
+                   elif test "${fuse_word}" = "73a"; then \
+                       env set fuse_word 740; \
+                   fi; \
```

Comment:
As said before, this whole machine specific knowledge could come from bitbake variables and not be directly encoded here.

Reply:
I'll close this because it's implicitly covered by another thread.